hipcheck/src/engine.rs

```diff
@@ -37,6 +37,7 @@ fn query(
 	};
 	// Initiate the query. If remote closed or we got our response immediately,
 	// return
+	println!("Querying {plugin}::{query} with key {key:?}");
```

Comment:
Whenever this code stops being experimental, these will need to become logging calls.